libraryValidations/Dotnet/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "profiles": {
+    "Dotnet": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "APP_CONFIG_VALIDATION_CONNECTION_STRING": "Endpoint=https://appconfig4ndakx7twshlm.azconfig.io;Id=A3jj;Secret=1WWpbi6jB5Bw9sYAvrHfSteqInR0UCZhwA2ynipVN1LcutGRncMyJQQJ99AJACYeBjFAArohAAACAZACIycQ"
```

Comment:
This file should be removed.

Reply:
Yes. Turns out we're not supposed to .gitignore this file- this is just not the right place for env variables. So this will not be automatically excluded- you should merge from main or delete it.